package.json

```diff
@@ -74,9 +75,10 @@
   ],
   "license": "MIT",
   "scripts": {
+    "cleanup": "gatsby",
```

Comment:
I assume you meant `gatsby clean` this is causing your build to fail without clean it just displays the cli for gatsby.